### DISCUSSION TITLE
How to get galaxy Brain badge?

### DISCUSSION THREAD

**Discussion by legendsstudioofficial** [community/community#136163] [Discussions]
### Select Topic Area

Question

### Body

How to get galaxy Brain badge?

**Comment by rakeshvellaisamy** [comment]
#1 Response

**Comment by HAMZA-420** [comment]
Click on my answer as marked complete then navigate to my profile

**Comment by vunguyenkhanh** [comment]
Yes!
Response #1

**Comment by LiteBrite82** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
